lib/embulk/version.rb

```diff
@@ -1,3 +1,3 @@
 module Embulk
-  VERSION = '0.8.18'
+  VERSION = org.embulk.EmbulkVersion::VERSION
```

Comment:
We may need to keep this until v2 for forward compatibility of `selfupdate`.

See also: https://github.com/embulk/embulk/blob/4c4d114e84173af81649fc60eab555253baa30a0/lib/embulk/command/embulk_selfupdate.rb#L64-L69